core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,552 @@
+		<xsl:text>	</xsl:text>
+		<xsl:value-of select="@elementID"/>
+		<xsl:text disable-output-escaping="yes">([InputValidator&lt;br/></xsl:text>
+		<xsl:value-of select="tokenize(@className, '\.')[last()]"/>
+		<xsl:text>])</xsl:text>
+		<xsl:text>&#10;</xsl:text>
+	</xsl:template>
+
+	<xsl:template match="outputValidator" mode="convertElements">
+		<xsl:if test="@errorHandling = 'true'">
+			<xsl:text>	style </xsl:text>
+			<xsl:value-of select="@elementID"/>
+			<xsl:text> stroke-dasharray: 4 4</xsl:text>
+			<xsl:text>&#10;</xsl:text>
+		</xsl:if>
+		<xsl:text>	</xsl:text>
+		<xsl:value-of select="@elementID"/>
+		<xsl:text disable-output-escaping="yes">([OutputValidator&lt;br/></xsl:text>
+		<xsl:value-of select="tokenize(@className, '\.')[last()]"/>
+		<xsl:text>])</xsl:text>
+		<xsl:text>&#10;</xsl:text>
+	</xsl:template>
+
+	<xsl:template match="inputWrapper" mode="convertElements">
+		<xsl:if test="@errorHandling = 'true'">
+			<xsl:text>	style </xsl:text>
+			<xsl:value-of select="@elementID"/>
+			<xsl:text> stroke-dasharray: 4 4</xsl:text>
+			<xsl:text>&#10;</xsl:text>
+		</xsl:if>
+		<xsl:text>	</xsl:text>
+		<xsl:value-of select="@elementID"/>
+		<xsl:text disable-output-escaping="yes">([InputWrapper&lt;br/></xsl:text>
+		<xsl:value-of select="tokenize(@className, '\.')[last()]"/>
+		<xsl:text>])</xsl:text>
+		<xsl:text>&#10;</xsl:text>
+	</xsl:template>
+
+	<xsl:template match="outputWrapper" mode="convertElements">
+		<xsl:if test="@errorHandling = 'true'">
+			<xsl:text>	style </xsl:text>
+			<xsl:value-of select="@elementID"/>
+			<xsl:text> stroke-dasharray: 4 4</xsl:text>
+			<xsl:text>&#10;</xsl:text>
+		</xsl:if>
+		<xsl:text>	</xsl:text>
+		<xsl:value-of select="@elementID"/>
+		<xsl:text disable-output-escaping="yes">([OutputWrapper&lt;br/></xsl:text>
+		<xsl:value-of select="tokenize(@className, '\.')[last()]"/>
+		<xsl:text>])</xsl:text>
+		<xsl:text>&#10;</xsl:text>
+	</xsl:template>
+
+	<xsl:template match="pipe" mode="convertElements">
+		<xsl:variable name="isSwitch" select="@className='nl.nn.adapterframework.pipes.XmlSwitch'
+											or@className='nl.nn.adapterframework.pipes.CompareIntegerPipe'
+											or@className='nl.nn.adapterframework.pipes.CompareStringPipe'
+											or@className='nl.nn.adapterframework.pipes.FilenameSwitch'
+											or@className='nl.nn.adapterframework.pipes.XmlIf'"/>
+		<xsl:if test="@errorHandling = 'true'">
+			<xsl:text>	style </xsl:text>
+			<xsl:value-of select="@elementID"/>
+			<xsl:text> stroke-dasharray: 4 4</xsl:text>
+			<xsl:text>&#10;</xsl:text>
+		</xsl:if>
+		<xsl:text>	</xsl:text>
+		<xsl:value-of select="@elementID"/>
+		<xsl:choose>
+			<xsl:when test="$isSwitch">{</xsl:when>
+			<xsl:otherwise>(</xsl:otherwise>
+		</xsl:choose>
+		<xsl:value-of select="@name"/>
+		<xsl:text disable-output-escaping="yes">&lt;br/></xsl:text>
+		<xsl:choose>
+			<xsl:when test="@className = 'nl.nn.adapterframework.pipes.GenericMessageSendingPipe'">
```

Comment:
De GenericMessageSendingPipe is deprecated, dit moet de SenderPipe zijn (en wat van andere pipes die een Sender hebben?)